### ISSUE TITLE
xharness summary incorrect `161 tests' device not found, 150 tests passed.`

### ISSUE THREAD

**Issue by spouliot** [xamarin/xamarin-macios#8103]
### Steps to Reproduce

unsure

https://github.com/xamarin/xamarin-macios/commit/21f2160063307387ff8d861d0c5be5c6420a412f#commitcomment-37797571

### Expected Behavior

Message is confusing and incorrect

### Actual Behavior

xharness complains that 32bits devices are not present
which is normal since that config should only be run on 64 bits devices

### Environment

master

**Comment by spouliot** [comment]
13.19.0.130 has the issue - https://github.com/xamarin/xamarin-macios/commit/51b6b707e44627961ab42db9968e0274a9a88f4b (edited) 

13.19.0.129 :fire: Tests failed catastrophically on iOS (no summary found)
13.19.0.128 :fire: Tests failed catastrophically on iOS (no summary found)
13.19.0.127 :fire: Tests failed catastrophically on iOS (no summary found)
13.19.0.126 :fire: Tests failed catastrophically on iOS (no summary found)
13.19.0.125 :fire: Tests failed catastrophically on iOS (no summary found)
13.19.0.124 :fire: Tests failed catastrophically on iOS (no summary found)
13.19.0.123 :fire: Tests failed catastrophically on iOS (no summary found)
13.19.0.122 :fire: Tests failed catastrophically on iOS (no summary found)
13.19.0.121 :fire: Tests failed catastrophically on iOS (no summary found)
13.19.0.120 - timed out while running the tests - but did not try to run on 32bits

.124 and .125 are xharness related

**Comment by spouliot** [comment]
All is fine but shows as failed
https://github.com/xamarin/xamarin-macios/commit/723ec1857c0bf3492d9a21899553e28ed67a5164

**Comment by mandel-macaque** [comment]
The issues not with xharness, but with the iOS pipeline. The pipeline has the following error: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3580476&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=fb1961db-d514-5215-fe81-a42cdc309f86&l=95 

where the grep cannot find the file. That hack was added by Vincent when we did not have the support for the labels in all active branches (we could have done better by checking the branch name, but that does not matter).

I have updated the pipeline to not do that grep, always use the ios64 label, which should fix the issue.
